CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Delta Chat Android Changelog
 
+## v1.52.0
+2025-01
+
+* new group consistency algorithm
```

Comment:
Not sure if this is too technical, most users won't understand what it means. Maybe it's possible to find sth everyone understands. In the general direction of "Avoid situations where some members of a group think that Alice is part of the group and others don't" but this is too long and also everyone will wonder "who is Alice??" - I can't think of a really good phrasing myself.

Reply:
"Avoid situations where some members of a group think that someone is part of the group and others don't" or "Avoid situations where the members of a group have differing infos on who is part of the group" could work

Reply:
otoh, "algorithm" has become a more well known word in main-stream media, ex "twitter/facebook filter algorithm" etc. and "group consistency" is clear and anything having a "new" label should be good/better right??? :rofl:

Reply:
some people might have never experienced these problems so mentioning it might work more as a negative point than positive "so this was actually that broken?"